float64[3] thrust
---
bool success
